.php_cs.dist

```diff
@@ -9,11 +18,17 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'array_syntax' => array('syntax' => 'long'),
+        'braces' => array('allow_single_line_closure' => true),
+        'heredoc_to_nowdoc' => false,
```

Comment:
I think this should still be applied (we have an exclude rule below for the file *needing* to use heredoc because it is about testing its support)

Reply:
What's the point? Why should we care?